pyproject.toml

```diff
@@ -40,6 +40,7 @@ azure-identity = {version = "^1.15.0", optional = true}
 azure-keyvault-secrets = {version = "^4.8.0", optional = true}
 google-cloud-kms = {version = "^2.21.3", optional = true}
 resend = {version = "^0.8.0", optional = true}
+pynacl = "^1.5.0"
```

Comment:
can you remove this please. @sha-ahammed this is not a requirement for the sdk. it seems proxy specific.

Reply:
> can you remove this please. @sha-ahammed this is not a requirement for the sdk. it seems proxy specific.

Should I add it as a dependency for `[tool.poetry.extras]` in `pyproject.toml`

Reply:
I have reverted the commit. Can we have this merged? @krrishdholakia ?